webui/react/src/utils/chart.ts

```diff
@@ -44,8 +44,8 @@ export const getNumericRange = (values: number[], forceRange = true): Range<numb
   }, [ Number.POSITIVE_INFINITY, Number.NEGATIVE_INFINITY ] as Range<number>);
 
   if (forceRange && range[0] === range[1]) {
-    range[0] = Math.floor(range[0]);
-    range[1] = Math.ceil(range[1]);
+    range[0] = Math.floor(range[0] - Number.EPSILON);
+    range[1] = Math.ceil(range[1] + Number.EPSILON);
```

Comment:
this makes it work as ended for whole numbers. otherwise it was broken.

Reply:
**chore:** can you add a test for this 🙏